robot_calibration/launch/joint_states_update.launch

```diff
@@ -2,10 +2,11 @@
    <!-- Load the urdf into the parameter server. -->
    <param name="robot_description" command= "cat $(find robot_calibration)/urdf/simple_2link_arm.urdf"/>
    
-   <rosparam file="$(find robot_calibration)/yaml/message_parameters.yaml"/>
+   <rosparam file="$(find robot_calibration)/state_update_parameters/state_update_parameters.yaml"/>
```

Comment:
You may want to use this same folder for storing param files for nodes that don't do state updates. Considering this, change the folder name from `state_update_parameters` to a more generic one e.g. config. The .yaml file name should indicate this is for the 2d setup for clarity. Rename to `2d_state_update_params.yaml`.